source/jormungandr/tests/departure_board_tests.py

```diff
@@ -355,3 +361,174 @@ def test_vj_comment(self):
+            eq_(get_not_null(resp_dt, 'date_time'), ref_st.dt)
+            eq_(get_not_null(resp_dt, 'links')[0]['id'], ref_st.vj)
+
+
+def check_departures(response, reference):
+    """
+    check the values in a departures
+    """
+    for (resp, ref) in itertools.izip_longest(response, reference):
+        eq_(get_not_null(get_not_null(resp, 'stop_point'), 'id'), ref.sp)
+        eq_(get_not_null(get_not_null(resp, 'route'), 'id'), ref.route)
+        eq_(get_not_null(get_not_null(resp, 'stop_date_time'), 'departure_date_time'), ref.dt)
+
+
+@dataset(["basic_schedule_test"])
+class TestSchedules(AbstractTestFixture):
+
+    def test_classic_stop_schedule(self):
+        """default is Realtime stopschedule"""
+        response = self.query_region("stop_points/S1/stop_schedules?from_datetime=20160101T080000")
+
+        stop_sched = response["stop_schedules"]
+        is_valid_stop_schedule(stop_sched, self.tester)
+
+        self.check_stop_schedule_rt_sol(stop_sched)
+
+    def test_stop_schedule_base_schedule(self):
+        response = self.query_region("stop_points/S1/stop_schedules?from_datetime=20160101T080000"
+                                     "&data_freshness=base_schedule")
+
+        stop_sched = response["stop_schedules"]
+        is_valid_stop_schedule(stop_sched, self.tester)
+
+        check_stop_schedule(stop_sched,
+                            [StopSchedule(sp='S1', route='A:0',
+                                          date_times=[SchedDT(dt='20160101T090000', vj='A:vj1'),
+                                                      SchedDT(dt='20160101T100000', vj='A:vj2'),
+                                                      SchedDT(dt='20160101T110000', vj='A:vj3')]),
+                             StopSchedule(sp='S1', route='B:1',
+                                          date_times=[SchedDT(dt='20160101T113000', vj='B:vj1')])])
+
+    @staticmethod
+    def check_stop_schedule_rt_sol(stop_sched):
+        check_stop_schedule(stop_sched,
+                            [StopSchedule(sp='S1', route='A:0',
+                                          date_times=[SchedDT(dt='20160101T090700',
+                                                              vj='A:vj1:modified:0:delay_vj1'),
+                                                      SchedDT(dt='20160101T100700',
+                                                              vj='A:vj2:modified:0:delay_vj2'),
+                                                      SchedDT(dt='20160101T110700',
+                                                              vj='A:vj3:modified:0:delay_vj3')]),
+                             StopSchedule(sp='S1', route='B:1',
+                                          date_times=[SchedDT(dt='20160101T113000', vj='B:vj1')])])
+
+    def test_stop_schedule_realtime(self):
+        """
+        test a stop schedule without a from_datetime parameter
```

Comment:
comment is not accurate I guess. You check that all is OK before testing without dt, do you?